spec/base_spec_helper.rb

```diff
@@ -4,14 +4,14 @@
 
 ENV["RAILS_ENV"] ||= 'test'
 
-require 'simplecov' if ENV["COVERAGE"]
-require 'simplecov-lcov'
+require 'simplecov' if ENV["COVERAGE"] || ENV["RAILS_ENV"]
+require 'simplecov-lcov' if ENV["COVERAGE"] || ENV["RAILS_ENV"]
```

Comment:
That's an odd condition. We set the rails env above and then it's always set. The condition is always true and therefore useless.

Reply:
Agree. Perhaps it would be better to remove  `if ENV["COVERAGE"] ` when requiring 'simplecov', I think we'd need it to run undercover locally.